laurent22/joplin
# Desktop: Fixes: Display fallback icons on a reasonable size on HTML notes

Fixes https://discourse.joplinapp.org/t/huge-black-arrows-in-html-emails/41498

# Summary

The issue the user reports happens when an HTML note (webclipper/email-to-note) is synchronized but not all resources have been downloaded yet. When that happens a fallback icon is rendered in the place of it.

The problem is that for the icon to be in a reasonable size, some CSS rules are applied.

The HTML note shouldn't need most of our CSS rules, so these basic rules are only applied to the Markdown notes.


# Testing

1. Use webclipper to save a page with a lot of images
2. Synchronize to a remote server
3. Remove local files (e.g.: delete `~/.config/joplindev-desktop/`)
4. Reopen the Desktop app
5. When the `Fetch Items` has finished, the note from the clipper should appear in the list
6. Stop the synchronisation by shutting down the server/shutting down the internet or observe the note while the resource is being downloaded
7. The icons should be shown in a reasonable size like the image below
![image](https://github.com/user-attachments/assets/147319af-74b8-492e-b23b-ed1f1004ab95)
